### PR TITLE
chp14: Following users

### PR DESCRIPTION
**Pull request by ngyihkai**
Ready

### CODE REVIEW

**Comment by ngyihkai** [comment]
Ready

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -7,3 +7,24 @@
...
+      <section class="micropost_form">
+        <%= render "shared/micropost_form" %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3>Micropost Feed</h3>
+      <%= render "shared/feed" %>
+    </div>
+  </div>
+<% else %>
```
Comment (huongnt-2545) [inline app/views/static_pages/home.html.erb]: remove this line

File: test/fixtures/relationships.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (huongnt-2545) [inline test/fixtures/relationships.yml]: delete this file

File: test/models/relationship_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/models/relationship_test.rb]: delete this file

**Comment by ngyihkai** [comment]
Ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**